### PR TITLE
Handle proportion field completeness and show important.

### PR DESCRIPTION
**Pull request by maxpeterson**

### CODE REVIEW

**Comment by maxpeterson** [comment]
@Minglee01 please review / merge

File: web/templates/twig-source/incuna-surveys/fields/base/checkbox.html [hunk 1]
```diff
@@ -1,8 +1,11 @@
-<div drf-form-field="to.fieldOptions" class="checkbox">
-    <div ng-repeat="choice in to.choices" class="checkable checkbox">
-        {% raw %}
-        <input class="checkbox-input" id="{{ to.autoId }}-{{ $index }}" type="checkbox" checklist-model="model[to.fieldSetId][options.key]" checklist-value="$index">
-        <label class="checkbox-label" for="{{ to.autoId }}-{{ $index }}" ng-bind="choice"></label>
-        {% endraw %}
+{% extends "./base.html" %}
+{% block feild %}
```
Comment (Minglee01) [inline web/templates/twig-source/incuna-surveys/fields/base/checkbox.html]: feild -> field

File: web/templates/twig-source/incuna-surveys/fields/base/checkbox.html [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
+    <div drf-form-field="to.fieldOptions" class="checkbox">
+        <div ng-repeat="choice in to.choices" class="checkable checkbox">
+            {% raw %}
+            <input class="checkbox-input" id="{{ to.autoId }}-{{ $index }}" type="checkbox" checklist-model="model[to.fieldSetId][options.key]" checklist-value="$index">
```
Comment (Minglee01) [inline web/templates/twig-source/incuna-surveys/fields/base/checkbox.html]: indent?

File: web/templates/twig-source/incuna-surveys/fields/base/free-text.html [hunk 3]
```diff
@@ -1,3 +1,6 @@
-<div drf-form-field="to.fieldOptions" class="text {% raw %}{{ model[to.fieldSetId][options.key] ? 'not-empty' : '' }}{% endraw %}" field-id="to.autoId">
-    <input type="text" class="text-input" id="{% raw %}{{ to.autoId }}{% endraw %}" ng-model="model[to.fieldSetId][options.key]" ng-required="to.fieldOptions.required">
-</div>
+{% extends "./base.html" %}
+{% block feild %}
```
Comment (Minglee01) [inline web/templates/twig-source/incuna-surveys/fields/base/free-text.html]: did u mean field

File: web/templates/twig-source/incuna-surveys/fields/base/radio.html [hunk 4]
```diff
@@ -1,8 +1,11 @@
...
-        <input class="radio-input" type="radio" id="{{ to.autoId }}-{{ $index }}" ng-value="$index" ng-model="model[to.fieldSetId][options.key]" ng-required="to.fieldOptions.required">
-        <label class="radio-label" for="{{ to.autoId }}-{{ $index }}" ng-bind="choice"></label>
-        {% endraw %}
+{% extends "./base.html" %}
+{% block field %}
+    <div drf-form-field="to.fieldOptions" class="radio">
+        <div ng-repeat="choice in to.choices" class="checkable radio">
+            {% raw %}
+            <input class="radio-input" type="radio" id="{{ to.autoId }}-{{ $index }}" ng-value="$index" ng-model="model[to.fieldSetId][options.key]" ng-required="to.fieldOptions.required">
+            <label class="radio-label" for="{{ to.autoId }}-{{ $index }}" ng-bind="choice"></label>
```
Comment (Minglee01) [inline web/templates/twig-source/incuna-surveys/fields/base/radio.html]: do we need indentation here?

**Comment by Minglee01** [comment]
looks ok but we need a changelog right?

File: web/templates/twig-source/incuna-surveys/fields/base/checkbox.html (hunk 2 above)
Comment (maxpeterson) [inline web/templates/twig-source/incuna-surveys/fields/base/checkbox.html]: I left the indent the same as before, not sure if we should indent for `{% raw %}` tags  .

Comment (Minglee01) [inline web/templates/twig-source/incuna-surveys/fields/base/checkbox.html]: i think we should indent for all tags. for consistency.

Comment (Minglee01) [inline web/templates/twig-source/incuna-surveys/fields/base/checkbox.html]: is there any reason why we shouldnt?

**Comment by maxpeterson** [comment]
Good point I will do that when I release it

**Pull request merged**